Dockerfile.gaia.prod

```diff
@@ -0,0 +1,87 @@
+FROM metabrainz/python:2.7
+
+ARG deploy_env
+
+# Install Dockerize
+ENV DOCKERIZE_VERSION v0.2.0
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+# Installing dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+                       build-essential \
+                       libyaml-dev \
+                       libfftw3-dev \
+                       libavcodec-dev \
+                       libavformat-dev \
+                       libavutil-dev \
+                       libavresample-dev \
+                       python-dev \
+                       python-numpy-dev \
+                       python-numpy \
+                       git \
+                       libsamplerate0-dev \
+                       libtag1-dev \
+                       libqt4-dev \
+                       swig2.0 \
+                       pkg-config \
+                       libpq-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# PostgreSQL client
+RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
+ENV PG_MAJOR 9.5
```

Comment:
Does it make sense to upgrade to a more recent version of postgres while we're doing all of this? A reload of the data is a big deal, might as well make it count, no?

Reply:
As per our discussion in IRC with @alastair, a move to PG 10 would be good.